tests/libdv/dv.go

```diff
@@ -151,6 +154,36 @@ func WithForceBindAnnotation() dvOption {
 	}
 }
 
+func WithDefaultInstancetype(name, kind string) dvOption {
+	return func(dv *v1beta1.DataVolume) {
+		if dv.Labels == nil {
+			dv.Labels = map[string]string{}
+		}
+		dv.Labels[instancetypeapi.DefaultInstancetypeLabel] = name
+		dv.Labels[instancetypeapi.DefaultInstancetypeKindLabel] = kind
+	}
+}
+
+func WithDefaultPreference(name, kind string) dvOption {
+	return func(dv *v1beta1.DataVolume) {
+		if dv.Labels == nil {
+			dv.Labels = map[string]string{}
+		}
+		dv.Labels[instancetypeapi.DefaultPreferenceLabel] = name
+		dv.Labels[instancetypeapi.DefaultPreferenceKindLabel] = kind
+	}
+}
+
+func WithDataVolumeSourceRef(name, namespace, kind string) dvOption {
```

Comment:
```suggestion
func WithDataVolumeSourceRef(namespace, name, kind string) dvOption {
```

Stay consistent with other options?

https://github.com/kubevirt/kubevirt/blob/51f66d67e52cdcd2f07d4a6df12d65eab423b117/tests/libdv/dv.go#L137

Reply:
It is a nit, but I also have hard time to ignore it :)
I would even go one step ahead and argue that the `kind` part should come before them all (including the prev commit).